physical/consul.go

```diff
@@ -577,3 +360,274 @@ func (c *ConsulLock) Value() (bool, string, error) {
 	value := string(pair.Value)
 	return held, value, nil
 }
+
+func (c *ConsulBackend) NotifyActiveStateChange() error {
+	select {
+	case c.notifyActiveCh <- notifyEvent{}:
+	default:
+		// NOTE: If this occurs Vault's active status could be out of
+		// sync with Consul until reconcileTimer expires.
+		c.logger.Printf("[WARN]: consul: Concurrent state change notify dropped")
```

Comment:
I don't think there is a need to warn, this is not really an issue, its just a rapid state change.